manifests/o/OpenJS/NodeJS/LTS/18.16.1/OpenJS.NodeJS.LTS.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: OpenJS.NodeJS.LTS
+PackageVersion: 18.16.1
+PackageLocale: en-US
+Publisher: Node.js Foundation
+PublisherUrl: https://nodejs.org
+PublisherSupportUrl: https://github.com/nodejs/help/issues
+Author: OpenJS Foundation
+PackageName: Node.js LTS
+PackageUrl: https://nodejs.org
+License: Copyright Joyent, Inc. and other Node contributors. All rights reserved.
+LicenseUrl: https://raw.githubusercontent.com/nodejs/node/v16.4.1/LICENSE

```

```suggestion
License: MIT
LicenseUrl: https://github.com/nodejs/node/blob/main/LICENSE
```
A copyright isn't a license